aspnetcore/security/app-secrets.md

```diff
@@ -69,12 +69,20 @@ In the preceding file paths, replace `<user_secrets_id>` with the `UserSecretsId
 
 Don't write code that depends on the location or format of data saved with the Secret Manager tool. These implementation details may change. For example, the secret values aren't encrypted, but could be in the future.
 
-::: moniker range="<= aspnetcore-2.0"
```

Comment:
I would still hide this section altogether for ASP.NET Core 2.1 and up. It will never apply to 2.1 users. Although you can build older runtimes with newer SDKs, you can't build newer runtimes with older SDKs.